scripts/db_migrator.py

```diff
@@ -882,15 +934,28 @@ def version_4_0_0(self):
         self.stateDB.set(self.stateDB.STATE_DB, 'FAST_RESTART_ENABLE_TABLE|system', 'enable', enable_state)
         self.set_version('version_4_0_1')
         return 'version_4_0_1'
-    
+
+
     def version_4_0_1(self):
         """
         Version 4_0_1.
-        This is the latest version for master branch
         """
         log.log_info('Handling version_4_0_1')
+        # Update cable length data and re-assign interface speed for T0 neighbor interfaces connected to EdgeZone Aggregators
+        self.update_edgezone_aggregator_config()
```

Comment:
cannot bump up the version here since this seems to be the latest for 202211 https://github.com/sonic-net/sonic-utilities/blob/202211/scripts/db_migrator.py#L869
Please check with Vaibhav if he has plans to bump up the version here

Reply:
Waiting on @vaibhavhd